oauth-jwt/jwt-auth-server/src/test/resources/Keycloack.postman_collection.json

```diff
@@ -0,0 +1,746 @@
+{
+    "variables": [],
+    "info": {
+        "name": "Keycloack",
```

Comment:
note the correct spelling is "Keycloak" -- let's correct it in the JSON content and filenames

Reply:
Fixed.